app/services/statements/statements_query.rb

```diff
@@ -0,0 +1,28 @@
+module Statements
```

Comment:
do we need the statement directory if we're going with the name `statements_query`?